code/game/machinery/doors/multi_tile.dm

```diff
@@ -25,37 +24,35 @@
 	opacity = 0
 	glass = 1
 	assembly_type = /obj/structure/door_assembly/multi_tile
+
 /obj/machinery/door/airlock/multi_tile/metal
 	name = "Airlock"
 	icon = 'icons/obj/doors/Door2x1metal.dmi'
 	assembly_type = /obj/structure/door_assembly/multi_tile
 
-
-	New()
-		..()
-
-		if(src.dir > 3)
-			f5 = new/obj/machinery/filler_object (src.loc)
-			f6 = new/obj/machinery/filler_object (get_step(src,EAST))
-		else
-			f5 = new/obj/machinery/filler_object (src.loc)
-			f6 = new/obj/machinery/filler_object (get_step(src,NORTH))
-		f5.density = 0
-		f6.density = 0
-		f5.set_opacity(opacity)
-		f6.set_opacity(opacity)
-
-	Destroy()
-		del f5
-		del f6
-		..()
+/obj/machinery/door/airlock/multi_tile/metal/New()
+	..()
+	if(src.dir > 3)
+		f5 = new/obj/machinery/filler_object (src.loc)
```

Comment:
Remove spaces between function call and parentheses. This comment also applies for next three lines.